kernel/device_manager/src/lib.rs

```diff
@@ -97,20 +97,20 @@ pub fn init(
         keyboard::init(ps2_controller.keyboard_ref(), key_producer)?;
         mouse::init(ps2_controller.mouse_ref(), mouse_producer)?;
     }
+    // Initialize/scan the PCI bus to discover PCI devices
+    for dev in pci::pci_device_iter()? {
+        debug!("Found PCI device: {:#X?}", dev);
+    }
```

Comment:
i can't comment on that line, but can you remove the outdated comment in L105 below?